tardis/adapters/sites/moab.py

```diff
@@ -8,15 +8,38 @@
 from ...utilities.staticmapping import StaticMapping
 from ...utilities.attributedict import convert_to_attribute_dict
 from ...utilities.executors.shellexecutor import ShellExecutor
+from tardis.utilities.asynccachemap import AsyncCacheMap
 
 from asyncio import TimeoutError
 from contextlib import contextmanager
 from functools import partial
 from datetime import datetime
+from collections import OrderedDict
```

Comment:
Is there any reason for using OrderedDict, here?

Reply:
OrderedDict is used in line 37 as datatype for the Moab resource status.

Reply:
And why?

Reply:
Ähm, good point. I will change it to a normal dictionary.